java/01-how-to-study.md

```diff
@@ -136,3 +136,22 @@ Provavelmente não irá conseguir, mas se você estava estudando algo em Maio,
 pode lembrar que anotou algo a respeito na semana do dia 20. Essas coisas nem
 sempre pegam de uma semana para outra: às vezes você precisa revisar coisas que
 viu há meses ou mesmo anos).
+
+Leia a documentação
+-----
+
+Você provavelmente algum dia já foi até o Google e procurou "How to <sua dúvida
+aqui>". Esta é uma prática muito comum quando precisamos fazer alguma coisa 
+rápida. Geralmente encontramos a resposta facilmente em sites como Stack Overflow
+ou similares e em alguns casos até conseguimos boas respostas. O problema é que
+algumas vezes a pessoa que te deu a resposta está te passando uma solução que 
+não é a melhor de todas (e ele não está fazendo isso por mal, provavelmente, mas
```

Comment:
Actually "best solution of all" are too strong words. "an inadequate solution" or "bad solution" could fit better.